client/web/src/util/constants.ts

```diff
@@ -12,4 +12,6 @@ export const enum EventName {
     TRY_CODY_MARKETPLACE = 'VSCodeMarketplace',
     TRY_CODY_WEB = 'TryCodyWeb',
     TRY_CODY_WEB_ONBOARDING_DISPLAYED = 'TryCodyWebOnboardingDisplayed',
+    TRY_CODY_SIGNUP_INITIATED = 'SignUpInitiated',
```

Comment:
I think we have to have cody in the event name for some tooling reasons 
```suggestion
    TRY_CODY_SIGNUP_INITIATED = 'CodySignUpInitiated',
```